deploy/Chart/templates/dashboard/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.dashboard.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dashboard-role
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: dashboard-role
+    app.kubernetes.io/part-of: radius
+rules:
+  - apiGroups: ['', 'api.ucp.dev']
+    resources: ['*']
+    verbs: ['*']
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I am wondering if dashboard requires cluster-scope permission. Ideally, it shouldn't. cc/ @rynowak

Reply:
looks like it does as of now. maybe this could be changed in the future.

Reply:
For calling into UCP it does require cluster-scoped. Kubernetes APIs look like:

**Cluster-scoped:**

```
/apis/<group>/<version>/....
```

**Namespace-scoped:**

```
/apis/<group>/<version>/namespaces/....
```

The UCP apis look like `/apis/api.ucp.dev/v1alpha3/<resourceID>`.